Bottom: 2a7bd6ec7958ed401d42c473c6933af37398b30d
Top:    64500518a30045be4bb61a809aa0aeb821e9bf88
Author: Robert Collins <robertc@robertcollins.net>
Date:   2013-03-06 22:34:29 +1300

Refresh of define-stream-result

---

diff --git a/NEWS b/NEWS
index 37b1f32..e27a06f 100644
--- a/NEWS
+++ b/NEWS
@@ -7,8 +7,9 @@ NEXT
 ~~~~
 
 A new sort of TestResult, the StreamResult has been added, as a prototype for
-a revised standard library test result API. The API is not stable yet, though
-we will endeavour to preserve compatibility for early adopters.
+a revised standard library test result API.  Expect this API to change.
+Although we will try to preserve compatibility for early adopters, it is
+experimental and we might need to break it.
 
 * The error message for setUp and tearDown upcall errors was broken on Python
   3.4. (Monty Taylor, Robert Collins, #1140688)
